src/routes/reference/melbands.svx

```diff
@@ -1,8 +1,86 @@
 ---
 title: MelBands
-blurb: Amplitude for a number of equally spread perceptual bands
+blurb: Magnitudes for a number of perceptually-evenly spaced spectral bands
 tags: 
     - descriptor
+    - mfcc
+    - mel
+    - perception
+    - spectra
 flair: reference
 category: Analyse Sound
----
\ No newline at end of file
+---
+
+<script>
+    import Image from '$lib/components/Image.svelte';
+    import YouTube from '$lib/components/YouTube.svelte';
+    import { Tabs, TabList, TabPanel, Tab } from '$lib/components/tabs/tabs';
+</script>
+
+`MelBands` returns a Mel-Frequency Spectrum which is a [histogram](https://en.wikipedia.org/wiki/Histogram) of FFT bin magnitudes bundled according to their relationship to the [Mel scale](https://en.wikipedia.org/wiki/Mel_scale). The Mel scale represents frequency space logarithmically, mimicking how humans perceive distance in pitch space. (The name "Mel" derives from the word "melody".)
+
+<YouTube
+url='UCLlVAj0PPY'
+/>
+
```

Comment:
Nice animation, perhaps explain what's happening in a very brief alt-texty way too